solidity/test/rewards/TestRewards.js

```diff
@@ -426,6 +426,18 @@ describe('Rewards', () => {
             expect(aliceBalance.toNumber()).to.equal(66666)
         })
 
+        it("lets closed keeps claim the rewards from multiple keeps correctly", async () => {
+            let timestamps = testValues.rewardTimestamps
+            await createKeeps(timestamps)
+            await rewards.setCloseTime(timestamps[2])
+            let rewardsReceivingKeeps = [0, 1]
+            await rewards.receiveRewards(rewardsReceivingKeeps, { from: aliceBeneficiary })
+            let aliceBalance = await token.balanceOf(aliceBeneficiary)
+            // Beneficiary will receive 200,000 / 3 = 66,666 per keep
+            // 66,666 * 2 = 133332 KEEP rewards total for being in 2 closed keeps
+            expect(aliceBalance.toNumber()).to.equal(133332)
+        })
```

Comment:
Following the style in `TestRewards.js` this test should be in a separate `describe` for `receiveRewards`.

Reply:
Done.